src/app/shared/model/series.model.ts

```diff
@@ -120,6 +120,9 @@ export class SeriesModel extends BaseModel implements HasUpload {
     data.title = this.title;
     data.descriptionMd = this.description;
     data.shortDescription = this.shortDescription;
+    if (this.changed('accountId')) {
+      data.set_account_uri = `api/v1/accounts/${this.accountId}`;
```

Comment:
should we be able to expand the existing account link and replace the id with the new account id? I'd feel better about doing that rather than hard coding anything for the API.